engines/bops_core/app/components/bops_core/status_detail_component.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module BopsCore
+  class StatusDetailComponent < GovukComponent::Base
+    renders_one :title
+    renders_one :body
+    renders_one :status
+
+    attr_reader :id, :open
+
+    def initialize(id: nil, open: false, classes: [], html_attributes: {})
+      @id = id
+      @open = open
+      super(classes:, html_attributes:)
+    end
+
+    def before_render
+      fail_missing_attributes
+    end
+
+    def call
```

Comment:
I think having this as a method rather than a separate template file is a definite improvement in this case 👍🏻